omniledger/darc/darc.go

```diff
@@ -414,20 +416,37 @@ func NewDarcSignature(signer *Signer, id ID, path []*Darc) (*Signature, error) {
 		return nil, errors.New("path missing")
 	}
 
-	hash, err := hashAll(id, darcsMsg(path))
+	// We create the message and then sign it, make sure the message is
```

Comment:
s/We c/C/

And we should discuss whether it's worth to discuss in what person we should write comments. Is it `We`, `It`, `I`, ...? I always struggle and probably change it according to the time of day...

Reply:
I usually just use "we", that's kind of the default when writing papers at least